llvm/test/DebugInfo/X86/line-0-no-is-stmt.ll

```diff
@@ -0,0 +1,33 @@
+; Test that line 0 records don't have is_stmt set
+; This tests the fix for LLVM issue #33870 (Bugzilla #34522)
+;
+; When scopeLine is 0, the initial location directive should not have is_stmt set.
+
+; RUN: %llc_dwarf -mtriple=x86_64-unknown-linux -O0 -filetype=obj < %s | llvm-dwarfdump --debug-line - | FileCheck %s
+
+; The line table entry for line 0 should exist but not have "is_stmt" in its Flags
+; CHECK: Address
+; CHECK: 0x{{[0-9a-f]+}} 0 0
+; CHECK-NOT: is_stmt
+; CHECK: end_sequence
```

Comment:
Can we CHECK: the entire line instead? I feel that will be clearer to understand visually